App.js

```
@@ -0,0 +1,604 @@
+
+  // FIXED: Cabbage varieties - only "cabbage" will trigger popup
+  'red cabbage': 'Red Cabbage', 'chinese cabbage': 'Chinese Cabbage',
+  'china cabbage': 'Chinese Cabbage', 'nappa cabbage': 'Chinese Cabbage',
+  'cabbage indian': 'Cabbage Indian', 'indian cabbage': 'Cabbage Indian',
+
+  // Asparagus - FIXED: imp/important mapping
+  'asparagus': 'Aspragus', 'aspargus': 'Aspragus', 'asparegus': 'Aspragus',
+  'local asparagus': 'Aspragus', 'regular asparagus': 'Aspragus',
+  'imported asparagus': 'Imp Aspragus', 'imp asparagus': 'Imp Aspragus',
+  'important asparagus': 'Imp Aspragus', 'asparagus important': 'Imp Aspragus','important aspargus': 'Imp Aspragus',
+  'asparagus imported': 'Imp Aspragus', 'asparagus imp': 'Imp Aspragus','asparagus imp': 'Imp Aspragus',
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)